README.md

```diff
@@ -1 +1,39 @@
-# example-osgi-portlet
\ No newline at end of file
+# Example OSGi portlet
+
+This application serves as a small example portlet that can be used in
+an OSGi-compliant ecosystem, in this case Liferay 7.
+
+## Prerequisites:
+
+- JDK11
+- Maven > 3
+- Internet connection to resolve dependencies
+
+
+## Local execution
+
+For development purposes, the application can be run as servlet in a
+local Jetty environment, without the need to set up and host a local
+OSGi environment.
+
+To run the application as servlet locally, check out the latest commit
+locally, and run the following command in your command line:
+
+```
+mvn clean -P no-liferay install jetty:run
```

Comment:
I think there is no need to install the jar of the portlet in your local maven repository:

```suggestion
mvn -P no-liferay clean package jetty:run
```

Reply:
If you only want to run it then you do not need to package it.
```suggestion
mvn clean jetty:run  -P no-liferay
```
NIT: place the profile last as suggested by the maven documentation https://maven.apache.org/guides/introduction/introduction-to-profiles.html